standard/lexical-structure.md

```diff
@@ -99,31 +105,33 @@ ASTERISK : '*' ;
 SLASH    : '/' ;
 ```
 
-Although these are lexer rules, these names are spelled in all-uppercase letters to distinguish them from ordinary lexer rule names.
+Although these are lexer rules, these names are spelled in all-uppercase letters to distinguish them from ordinary lexer rule names. (*Note*: These convenience rules are exceptions to the usual practice of not providing explicit token names for tokens defined by literal strings. *end note*)
 
-The *Input* production defines the lexical structure of a C# compilation unit.
+The *input* production defines the lexical structure of a C# compilation unit.
 
 ```ANTLR
-Input
-    : Input_Section?
+input
+    : input_section?
     ;
 
-Input_Section
-    : Input_Section_Part+
+input_section
+    : input_section_part+
     ;
 
-Input_Section_Part
-    : Input_Element* New_Line
-    | Pp_Directive
+input_section_part
+    : input_element* New_Line
+    | PP_Directive
     ;
 
-Input_Element
+input_element
     : Whitespace
     | Comment
-    | Token
+    | token
     ;
 ```
 
+> *Note*: The above grammar is described by ANTLR parsing rules, it defines the lexical structure of a C# compilation unit and not lexical tokens. *end note*
```

Comment:
Really this grammar is not about tokens (lexical) but about arrangements of tokens (parser). A lexer might implement them to check the structure of the input but there is no need and in ANTLR they are simply ignored as there is no path from the start symbol of the parser grammar to them. However they are the only place that *PP_Directive* is referenced, but that is no ANTLR-ised yet...